### ISSUE TITLE
Online Archive Not Updating and Utility Files Not Completing Properly

### ISSUE THREAD

**Issue by blab052801** [Ackater/writing.com-archival#57]
The online archive hasn't been updating for almost three months and the downloadable utility files do not function properly.

To elaborate, the lxml version listed in requirments.txt is no longer available (updating the version listed fixes any errors thrown as a result) and upon running "./run.py get" the program will create a folder for the story to be archived, but will exit without populating said folder wit any files.

**Comment by Ackater** [comment]
Sorry I've been busy and with their latest anti-bot measures I have not been motivated to bypass. The interactives I watch don't have too many updates either. I do want to get back to it in the future

**Comment by JustAKat** [comment]
Oh, was this why the archives now show nothing? Hope they aren't lost to time then, your archives were a godsend

**Comment by Ackater** [comment]
Nope, that was a DNS issue. Forgot to renew the domain, it should be working now.

**Comment by cryo420** [comment]
i still dont get the list, it might not be dns
